static-analysis/action.yaml

```diff
@@ -1,10 +1,6 @@
 name: Run static code analysis
 description: GitHub Action that scans code changes being made and posts security findings as comments on pull requests.
 inputs:
-  checkout-repo:
```

Comment:
This makes this a breaking change technically, although if the input was not in use then it should be ~ok